run/default-config.yaml

```diff
@@ -0,0 +1,4 @@
+# Default config -- Run MongoDB locally
+mongo_url: "mongodb://10.1.15.231:27017"
+mongo_database: traces2
```

Comment:
let's call the database `jaeger-tracing` or just `tracing`. not sure which one you prefer.

Reply:
I have renamed it to jaeger-tracing